### PR TITLE
fix(core): Fix workflow activation with history and workflow history for EE (no-changelog)

### PR DESCRIPTION
**Pull request by valya**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: packages/cli/src/workflows/workflowHistory/workflowHistory.service.ee.ts [hunk 1]
```diff
@@ -66,13 +67,20 @@ export class WorkflowHistoryService {
 
 	async saveVersion(user: User, workflow: WorkflowEntity, workflowId: string) {
 		if (isWorkflowHistoryEnabled()) {
```
Comment (krynble) [inline packages/cli/src/workflows/workflowHistory/workflowHistory.service.ee.ts:69]: Feels a bit odd to me to have this check here for enablement and the caller checking for licensed. Maybe it makes sense to put both in the same place, preferably inside the function.

**Review by krynble — DISMISSED** [review]
LGTM, just left one comment

**Review by krynble — APPROVED** [review]
LGTM

**Comment by janober** [comment]
Got released with [`n8n@1.14.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.14.0)

**Pull request merged**